src/node/waveshaper.rs

```diff
@@ -256,6 +256,122 @@ struct RendererConfig {
     receiver: Receiver<CurveMessage>,
 }
 
+const DEFAULT_CHUNK_SIZE: usize = 128;
```

Comment:
If you have chosen this constant to correspond to the default render quantum size, I'd rather have you import `crate::RENDER_QUANTUM_SIZE`. It will make the migration easier for #217